src/components/Form.js

```diff
@@ -0,0 +1,60 @@
+import { toBeRequired } from "@testing-library/jest-dom/matchers";
+import React, { useState } from "react";
+
+import Users from './Users';
+import FormInput from "./FormInput";
+
+const email = 'digitalAcademy@gmail.com';
+const pass = 'iLoveReact123';
+
+
+
+const Form = () => {
+  const [user, setUser] = useState({ email: "", password: "" });
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  
+  const disabled = user.email !== email || user.password !== pass;
+
+  const submitHandler = (e) => {
+    setIsLoggedIn(true);
+  };
+
+  if (isLoggedIn) {
+    return (
+      <Users />
+    );
+  }
+
+  const settings = [
```

Comment:
ეს მასივი კომპონენტის გარეთ უნდა გეწეროს

Reply:
is done